exercises/04-reading-raw-data.py

```diff
@@ -0,0 +1,46 @@
+'''
+  Author: Jorge Diaz Jr
+  Exercise 4
+  Pt1. Reading files into an array
+'''
+import numpy as np 
+import os
+
+directory_string = "/home/jdiaz/projects/data-monitoring/data/iss_sample_data"
+
+'''
+going to store each file name in a list
+index 2 indicates that we only want filenames and no directory path or directory names
+'next' grabs the next value in iterator 
+'''
+files_only = next(os.walk(directory_string))[2] 
```

Comment:
Awesome! Looks great!
another option (slight change from your code, not too much):
```py3
contents = os.listdir(DIR)
txt_filenames = [filename for filename in contents if os.path.isfile(filename) and filename.endswith(".txt")]
raw_filenames = [filename for filename in contents if os.path.isfile(filename) and not filename.endswith(".txt")]

```